src/tests/ftest/util/soak_utils.py

```diff
@@ -320,20 +320,22 @@ def wait_for_pool_rebuild(self, pool, name):
     """
     rebuild_status = False
     self.log.info("<<Wait for %s rebuild on %s>> at %s", name, pool.identifier, time.ctime())
-    self.dmg_command.server_set_logmasks("DEBUG", raise_exception=False)
+    if self.enable_rebuild_logmasks:
+        self.dmg_command.server_set_logmasks("DEBUG", raise_exception=False)
```

Comment:
FYI there is already logic for this per-pool:

https://github.com/daos-stack/daos/blob/bf0bedbc4f1a75ec42c057d56418dbe826b818eb/src/tests/ftest/pool/create_capacity.yaml#L50

And then used like this
https://github.com/daos-stack/daos/blob/bf0bedbc4f1a75ec42c057d56418dbe826b818eb/src/tests/ftest/util/test_utils_pool.py#L421
Where `self` is the pool object.


Do you only need this disabled for rebuild, or should it be disabled for other pool operations too?